go/pkg/pwcompose/compose.go

```diff
@@ -169,12 +169,15 @@ func Prepare(challengeDir string, prefix string, noPush bool, version string, lo
 	return nil
 }
 
-func Up(preparedCompose string, instanceKey string, logger *zap.Logger) error {
-	logger.Debug(
-		"up",
-		zap.String("compose", preparedCompose),
-		zap.String("instance-key", instanceKey),
-	)
+func Up(
+	ctx context.Context,
+	preparedCompose string,
+	instanceKey string,
+	forceRecreate bool,
```

Comment:
if you prefer, you can put all the non-necessary vars in a new `UpOpts struct` (`instanceKey`, `forceRecreate`)

Reply:
will do only if we have more than 2 args

Reply:
same for me